aikau/src/main/resources/alfresco/layout/AlfStackContainer.js

```diff
@@ -0,0 +1,558 @@
+            doLayout: this.doLayout
+         }, this.containerNode);
+         this.stackContainerWidget.watch("selectedChildWidget", lang.hitch(this, "_paneChanged"));
+
+         // Setup child widgets and startup()
+         if (this.widgets)
+         {
+            // By default we want to ensure that we don't unnecessarily process widgets for 
+            // panes that are not immediately visible. Therefore unless specifically requested 
+            // in the configuration to be the selected pane or to render immediately then
+            // we'll mark them all as delayed processing. If no pane is marked as selected then
+            // we'll ensure that the first pane is both selected and will be immediately rendered
+            var paneSelected = false;
+            array.forEach(this.widgets, function(widget) {
+               if (widget.delayProcessing !== false && !widget.selected)
+               {
+                  widget.delayProcessing = true;
+               }
+               paneSelected = paneSelected || widget.selected;
+            });
+            if (this.widgets.length && !paneSelected)
+            {
+               this.widgets[0].selected = true;
+               this.widgets[0].delayProcessing = false;
+            }
+
+            // Now add panes for each widget...
+            array.forEach(this.widgets, lang.hitch(this, "addWidget"));
+         }
+         this.stackContainerWidget.startup();
+
+         // Subscribe to some optional topics
+         if (this.paneSelectionTopic != null)
+         {
+            this.alfSubscribe(this.paneSelectionTopic, lang.hitch(this, "paneSelect"));
+         }
+         if (this.paneAdditionTopic != null)
+         {
+            this.alfSubscribe(this.paneAdditionTopic, lang.hitch(this, "paneAdd"));
+         }
+         if (this.paneDeletionTopic != null)
+         {
+            this.alfSubscribe(this.paneDeletionTopic, lang.hitch(this, "paneDelete"));
+         }
+         if (this.paneSelectionHashVar != null)
+         {
+            this.alfSubscribe(this.hashChangeTopic, lang.hitch(this, this.onHashChanged));
+
+            // Initialise the view by invoking onHashChanged, as though the browser had done so
+            this.onHashChanged(ioQuery.queryToObject(hash()));
+         }
+      },
+
+      /**
+       * This function adds widgets to the StackContainer widget
+       * 
+       * @instance
+       * @param {object} widget The widget to add
+       * @param {integer} index The index of the required pane position
+       */
+      addWidget: function alfresco_layout_AlfStackContainer__addWidget(widget, index) {
+
+         // Create a domNode and ContentPane
+         var domNode = domConstruct.create("div", {}),
+             cp = new ContentPane();
+
+         // Add content to the ContentPane
+         if(widget.content && typeof widget.content === "string")
+         {
+            cp.set('content', widget.content);
```

Comment:
Single-quotes not permitted in Aikau any more. Please run JSHint on all changed files and fix any resultant bugs. The `.jshintrc` config file in the Aikau project should be picked up automatically by most JSHint runtimes.